content/applications/finance/accounting/payments/follow_up.rst

```diff
@@ -14,44 +14,61 @@ Configuration
 =============
 
 To configure a :guilabel:`Follow-Up Action`, go to :menuselection:`Accounting --> Configuration -->
-Follow-up Levels`. Several follow-up actions are available by default, and you can customize them
-any way you want or create new ones to trigger the following actions:
+Follow-up Levels`. Several follow-up actions are available by default under the
+:guilabel:`Notification`, but you can create new ones if required, and can change the **name** and
+the **number of days** after which the follow-up actions are triggered. The follow-up
+:guilabel:`Actions` available are:
 
-- Send an email;
-- :ref:`Send an SMS message <pricing/pricing_and_faq>`;
-- Print a letter;
-- :ref:`Send a letter <customer_invoices/snailmail>`;
-- Manual action (creates a task).
+- :guilabel:`Send Email`;
+- :ref:`Send a Letter <customer_invoices/snailmail>`;
+- :ref:`Send an SMS message <pricing/pricing_and_faq>`.
 
-It is also possible to automatically send a reminder by enabling the :guilabel:`Auto Execute`
-option, within a specific :guilabel:`Follow-Up Action`.
+You can use a pre-filled template for your messages by selecting a :guilabel:`Content Template`. To
+change the template used, hover over the the field and click the :guilabel:`-->`. If enabled, SMS
+messages have a specific :guilabel:`Sms Template` field.
+
+It is possible to automatically send a reminder by enabling the :guilabel:`Automatic` option, and
+attaching the *open* invoice(s) by enabling :guilabel:`Attach Invoices`, within a specific follow-up
+action.
+
+By clicking on the :guilabel:`Activity` tab, it's possible to schedule activities (tasks). That way,
+when the follow-up is triggered, an activity is automatically scheduled. To do so, enable
```

Comment:
By clicking on the :guilabel:`Activity` tab, scheduling activities (tasks) is possible. That way, an activity is automatically scheduled when the follow-up is triggered.